projects/backbone/app/jobs/weekly_digest_job.rb

```diff
@@ -71,7 +71,7 @@ def perform(channel: "#general")
 
   def more_or_less_percentage_in_words(percentage)
     if percentage < 0
-      "#{percentage}% less than"
+      "#{-percentage}% less than"
     else
       "#{percentage}% more than"
```

Comment:
Maybe writing it differently it's more evident if things are going down or up, something like:
```
  def more_or_less_percentage_in_words(percentage)
    "#{percentage}% with respect to"
  end
```

Reply:
We could even go fancier and add some emoji like ⬆️ /📈 and ⬇️ /📉

Reply:
Love the idea of emojis!